### ISSUE TITLE
MEI version?

### ISSUE THREAD

**Issue by rettinghaus** [pfefferniels/probstuecke-digital#5]
For some files MEI3 is use, others are in MEI4. If there isn't a good reason for this, all files should use the same version.

**Comment by pfefferniels** [comment]
Right, I think there's no reason not to have them all on version 4. I'll change that.
